src/azure-cli/azure/cli/command_modules/resource/_packing_engine.py

```diff
@@ -35,10 +35,7 @@ def process_template(template, preserve_order=True, file_path=None):
     template = re.sub(r'(^[\t ]*//[\s\S]*?\n)|(^[\t ]*/\*{1,2}[\s\S]*?\*/)', '', template, flags=re.M)
 
     # In order to solve the package conflict introduced by jsmin, the jsmin code is referenced into json_min
-    minified = json_min(template)
-
-    # Remove extra spaces, compress multiline string(s)
-    result = re.sub(r'\s\s+', ' ', minified, flags=re.DOTALL)
```

Comment:
Actually, the `az ts create` command is owned by service team. This line of code was introduced by PR https://github.com/Azure/azure-cli/pull/29437/files
+ARM team @detienne20 Could you confirm the purpose of adding this line of code earlier, and whether deleting it is acceptable and without regression?

Reply:
This is a breaking change. My suggestion is to instead preserve the original intention:
Replace `\n` with `\\n`
Replace `\r` with `\\r`

Reply:
Hi @anthony-c-martin, thanks for your review. 
The "az ts" related commands are actually owned by service team, and Azure CLI team is just helping fix this issue. Could you please raise another PR to fix it? I'll close this PR. Thanks!

Reply:
@yanzhudd - submitted #29623 for this